content/packit-as-a-service.md

```diff
@@ -65,3 +65,15 @@ pull request in COPR repository and reports back COPR build status to the pull r
 An example of Packit Service pull request:
 
 ![Packit service pull request](/packit-service-pr.png)
+
+## FAQ
+
+### Can I use packit service as soon as I install it into my repository?
+
+If you installed Packit-as-a-service, we have to approve you first.
+This is mainly because of security reasons and approval can take up to 7 days.
+
+### Can I use packit service for any GitHub repository?
+
+As packit allows you to build your PRs in COPR, by using Packit-as-a-service you agree to comply with: [COPR rules](https://docs.pagure.org/copr.copr/user_documentation.html#what-i-can-build-in-copr).
+If any of these points is violated, the user will be banned from using Packit-as-a-service.
```

Comment:
I would personally improve the wording a bit and act more friendly. So instead of saying

    If you installed Packit-as-a-service, we have to approve you first.

I would say something like:

    Thanks for your interest in Packit Service! Before you'd be able start using it, we need to put you on a whitelist: feel free to get in touch with us or we'll reach out to you once we add you to whitelist.

You don't need to take my precise words, but adding a little bit of :heart: to the text would make us look more welcoming and friendly.

Reply:
To be able to start using the service,....
Anyway, I don't understand @TomasTomecek sentence much. Will they be whitelisted automatically? Why should they be able to reach out to us? To speed it up?

Reply:
Okay, so let's leave Rado's text then, that's super-clear. In the end we're not native speakers so why should we act like we are.

Reply:
Or in the other words, I'll send a PR if I figure out improvements.